Utils/ReadDynamicSizeArray.java

```diff
@@ -0,0 +1,30 @@
+package Utils;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class ReadDynamicSizeArray {
+    private int numberOfLinesInArray;
+
+    private int numberOfElementsPerLine;
+
+    private int element;
+
+    public ArrayList[] readDynamicArray() {
+        Scanner scanner = new Scanner(System.in);
+        ArrayList[] arrayList = new ArrayList[20000];
+        System.out.println("Give the number of lines in array: ");
+        numberOfLinesInArray = scanner.nextInt();
+        for (int i = 0; i < numberOfLinesInArray; i++) {
+            arrayList[i] = new ArrayList();
+            System.out.println("Give the number of elements for the line: ");
+            numberOfElementsPerLine = scanner.nextInt();
```

Comment:
Create additional option for bulk data.
Sample input
5
5 41 77 74 22 44
1 12
4 37 34 36 52
0
3 20 22 33
5
1 3
3 4
3 1
4 3
5 5
Sample Output

74
52
37
ERROR!
ERROR!

If you need additional explanations, let's discuss